### ISSUE TITLE
https://pennydreadfulmagic.com/seasons/21/metagame/tournament/ gives a 500

### ISSUE THREAD

**Issue by vorpal-buildbot** [silasary/Penny-Dreadful-Discord-Bot#11143]
Reported on Discord by bakert#2193

**Comment by bakert** [comment]
```
Traceback (most recent call last):
  File "/home/pdadmin/decksite/.venv/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pdadmin/decksite/.venv/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pdadmin/decksite/.venv/lib/python3.10/site-packages/flask_restx/api.py", line 674, in error_router
    return original_handler(e)
  File "/home/pdadmin/decksite/.venv/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pdadmin/decksite/.venv/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/pdadmin/decksite/./decksite/cache.py", line 80, in decorated_function
    response = make_response(f(*args, **kwargs))
  File "/home/pdadmin/decksite/./decksite/controllers/metagame.py", line 42, in metagame
    view = Metagame(disjoint_archetypes, tournament_only=tournament_only, key_cards=key_cards)
  File "/home/pdadmin/decksite/./decksite/views/metagame.py", line 42, in __init__
    a.lower_bound, a.upper_bound = confidence_interval(float(a.win_percent) / 100.0, a.num_matches)
ValueError: could not convert string to float: ''
``

**Comment by bakert** [comment]
Interestingly this does not open a ticket at perf-reports repo which I believe 500 errors are supposed to. Did that bit rot away or get disabled at some point?

**Comment by bakert** [comment]
I fixed this a few days ago in a76d60c but apparently forgot to close the ticket.
